scripts/make_raja_branch_from_fork_pr.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+###############################################################################
+# Help                                                                         
+###############################################################################
+Help()
+{
+   # Display Help
+   echo "Syntax: make_raja_branch_from_fork_pr [-h|num]"
+   echo "options:"
+   echo "-h     Print this help usage message."
+   echo "num   PR number."
+   echo
+}
+
+###############################################################################
+# Process the input options. 
+###############################################################################
+# Get the options
+while getopts ":h" option; do
+   case $option in
+      h) # display Help
+         Help
+         exit;;
+     \?) # incorrect option
+         echo "Error: Invalid option"
+         exit;;
+   esac
+done
+
+echo "Making RAJA branch from branch on fork for PR $1";
+
+echo "When script exits, you will be on branch pr-from-fork/$1 in your local RAJA repo."
+
+echo "Do 'git push -u' to push the new branch to the main RAJA repo."
```

Comment:
I find that when a script tells the user to run a command, many people (myself included) will copy-paste it and try to run it.  Can this be `git push -u origin pr-from-fork/$1` (adjusted accordingly if a custom remote option is added per @trws's comment)?

Reply:
I'd probably add the full command, with the remote and branch name since that command can (depending on config) push *all* branches without it, and then either:

1. ask y/n with default n and just execute it for the user, and/or
2. offer a flag to do the push

I'd actually be happy to add some stuff in here to auto-generate the PR if gh is available and the user approves too if that's desirable.

Reply:
@trws I pushed some changes to clarify the script usage and make the output cleaner. Feel free to modify or add more to cover more use cases you have in mind.